src/mon/OSDMonitor.cc

```diff
@@ -80,7 +80,9 @@ OSDMonitor::OSDMonitor(CephContext *cct, Monitor *mn, Paxos *p, const string& se
    full_osd_cache(g_conf->mon_osd_cache_size),
    thrash_map(0), thrash_last_up_osd(-1),
    op_tracker(cct, true, 1)
-{}
+{
```

Comment:
@renhwztetecs Please keep PR title within 50 characters
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits